### DISCUSSION TITLE
ISIN code is not supported

### DISCUSSION THREAD

**Discussion by ahmetavc** [ghostfolio/ghostfolio#2728] [Q&A]
Hey, does ghostfolio only support USA related ISIN codes?

I am trying to add activity with TRMTCDWWWWW8, and is not supported.

**Comment by dtslvr** [comment]
Hi @ahmetavc

Ghostfolio does indeed support ISIN codes from outside the USA. However, please note that if a specific ISIN code is not present on _Yahoo Finance_, it won't be accessible in Ghostfolio either. This is because the market data is sourced from _Yahoo Finance_, especially in the self-hosted version.

**Comment by 0a1b** [comment]
https://help.yahoo.com/kb/finance-for-web/sln24383.html

Yahoo in itself relies on two providers. Since i had a similar problem i wrote morning star today. Lets see.

**Comment by colorcube** [comment]
I tried a few European WKN and ISIN on self hosted and nothing was found (on yahoo's website neither).
I can't try it on the demo website because adding something is disabled.

Does it work better in the premium version? Why? Paid Service?

**Reply by dtslvr** [reply]
> Does it work better in the premium version? Why? Paid Service?

Premium user get access to a professional data provider. You can create an account via _Get Started_ and apply for a free trial.

**Comment by je71175** [comment]
can we manually or otherwise set prices for funds not published on Yahoo ?  Is there a work around with the Premium Version ?

**Reply by dtslvr** [reply]
Yes, you could track custom assets not published by data providers by adding an activity of type _Valuables_, which allows for manual historical market data (experimental).
